CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 IMPROVEMENTS:
  * agent: Add `BlockQueryWaitTime` config option for Nomad API connectivity [[GH-755](https://github.com/hashicorp/nomad-autoscaler/pull/755)]
 
+BUG FIXES:
+ * agent: Fixed a bug that caused the agent to panic when trying to evaluate a policy with a `check` block missing a `strategy`  [[GH-813](https://github.com/hashicorp/nomad-autoscaler/pull/813)]
```

Comment:
Do we want something after `strategy` as it sounds like the sentence cuts short?

Reply:
Ops, I probably ran out of juice mid typing 😅